docs/xxgloballockreservation.md

```diff
@@ -0,0 +1,101 @@
+<!--
+* Copyright (c) 2017, 2020 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:\[+|-\]GlobalLockReservation
+
+**(Power only)**
+
+The `-XX:+GlobalLockReservation` option enables an optimization targeted towards more efficient handling of locking and unlocking Java&trade; objects.
+
+## Syntax
+
+        -XX:[+|-]GlobalLockReservation
+        -XX:+GlobalLockReservation:<parameter>
+
+| Setting                    | Effect | Default                                                                        |
+|----------------------------|--------|:------------------------------------------------------------------------------:|
+|`-XX:+GlobalLockReservation`| Enable |                                                                                |
+|`-XX:-GlobalLockReservation`| Disable| <i class="fa fa-check" aria-hidden="true"></i><span class="sr-only">yes</span> |
+
+This optimization is targeted towards applications with lots of uncontended locked objects that are being locked just to be safe. When enabled, heuristics are used to try and determine when an object will be exclusively locked by a single thread so that faster, more specialized code can be used for locking the object. Negative performance side effects may occur if the heuristics used get things wrong but functional correctness should still be maintained.
```

Comment:
I think the last sentence here could be re-worded for clarity (I don't think we need to talk about functional correctness - this is implied by the performance statement):

If the heuristics incorrectly identify an object as a target for processing, performance might be adversely affected.

Reply:
I'll make this change as well.